### ISSUE TITLE
Add error handling for save/load

### ISSUE THREAD

**Issue by robinlavallee** [guibec/rpgcraft#77]
No error handling is done at all, plus it's only integrated in the Debug menu.

**Comment by jstine35** [comment]
K most of those should be handled separately, from an architectural standpoint.  Some platforms and storage media have built in redundancy (Xbox) or make typical error checking difficult or ineffective (ps4). And cloud has a different ideal experience, since in that case a failure should re-download from online storage.

On pc people are pretty much expected to backup and restore manually.  “Open explorer window here” is sufficient, especially if using proper non-exclusive D3D mode.

In none of these scenarios does the game’s internal save data APIs really need to care.

**Comment by robinlavallee** [comment]
Yes, except right now the save code doesn't even handle exceptions, so it will just silently fail. :P So at the very least, it needs to return true/false. ;)
